draft-ietf-tls-esni.md

```diff
@@ -1656,6 +1656,31 @@ important ideas and contributions.
 
 Issue and pull request numbers are listed with a leading octothorp.
 
+## Since draft-ietf-tls-esni-10
+
+- Make HRR confirmation and ECH acceptance explicit (#422, #423)
+
+- Relax computation of the acceptance signal (#420, #449)
+
+- Fix ClientHelloOuterAAD generation (#438, #442)
```

Comment:
Maybe s/Fix/Simplify/? The old version wasn't broken. Just this new one's hopefully simpler.

Reply:
Yep, that's better!